less/index.less

```diff
@@ -53,7 +53,183 @@ table {
     box-sizing: border-box;
 }
 
+//variables//
+@raleway: 'Raleway', sans-serif;
+@main-color: #E9B8B1;
+@gray-border: #dddddd;
+
+//mixins//
+.section-content() {
+	display: flex;
+	justify-content: space-between;
+}
+
+.subheader() {
+	font-size: 1.8rem;
+	font-weight: bold;
+	text-decoration: underline;
+}
+
 html, body {
-    height: 100%;
+		height: 100%;
+		font-size: 62.5%;
+		margin: 10px 2rem;
+		font-family: @raleway
+}
+.home {
+	font-size: 1.6rem;
+}
+
+.nav {
+	display: flex;
+	justify-content: flex-end;
+
+	@media( max-width: 500px) {
+	justify-content: center;
+	}	
+
+	a {
+		text-decoration: none;
+		color: black;
+		font-size: 1.8rem;
+		margin-left: 1rem;
+	}
+}
+
+a:hover {
```

Comment:
You can nest this inside your anchor selector on line 91 as `&:hover { ...... }`